exercise-2/admin-generators.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="UTF-8">
+	<title>admin-info</title>
+	 <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
```

Comment:
would prefer not to use bootstrap just yet as we are learning how to implement CSS

Reply:
Sorry Eric, actually i was trying out different types of styles but i forgot to remove the link tag later on while uploading the file.